cpp/tests/binaryop/assert-binops.h

```diff
@@ -69,6 +69,19 @@ struct NearEqualComparator {
   }
 };
 
+template <typename TypeLhs, typename ScalarType>
+TypeLhs scalar_host_value(cudf::scalar const& lhs)
+{
+  auto sclr   = static_cast<ScalarType const&>(lhs);
+  auto stream = cudf::get_default_stream();
+  if constexpr (std::is_same_v<ScalarType, cudf::string_scalar>)
+    return sclr.to_string(stream);
+  else if constexpr (std::is_same_v<ScalarType, cudf::fixed_point_scalar<TypeLhs>>)
+    return sclr.fixed_point_value(stream);
+  else
+    return sclr.value(stream);
```

Comment:
Please always use braces for readability.
```suggestion
  if constexpr (std::is_same_v<ScalarType, cudf::string_scalar>) {
    return sclr.to_string(stream);
  } else if constexpr (std::is_same_v<ScalarType, cudf::fixed_point_scalar<TypeLhs>>) {
    return sclr.fixed_point_value(stream);
  } else {
    return sclr.value(stream);
  }
```

Reply:
Addressed